docs/advanced-topics/checker-workflows.rst

```diff
@@ -120,43 +122,56 @@ is important that the checker workflow produce consistent exit codes. We
 require using an exit code of 0 for success and an exit code of not 0
 for failures.
 
-We also recommend producing the following two output files containing
-the stdout and stderr respectively:
-
-* log.stdout
-* log.stderr
+We also recommend producing the stderr and stdout logs. The exact filenames
+of these logs will vary depending on the platform and the workflow language. In
+CWL, they are typically referred to as log.stderr and log.stdout. WDL executors on
+the other hand tend to name these files stderr and stdout with no extension.
 
 Note on CLI usage
 ~~~~~~~~~~~~~~~~~
 
-For existing dockstore commands (tools and workflows), entry refers to
+As mentioned before, the term "entry" will be used as a generic term to refer to both tools
+and workflows. For existing dockstore commands (tools and workflows), entry refers to
 the path of a specific tool or workflow. For checker workflows, entry
 refers to the path of the original entry. It does not refer to the
 checker workflow's path.
 
-Adding a checker workflow
--------------------------
+Creating a checker workflow
+---------------------------
+Checker workflows are generally easy to create. First of all, if you wish
+to create a workflow that is based on comparison to truth files and/or truth
```

Comment:
Is truth files a known term? Like genuinely asking. I've never head of it and it kind of confused me.

Reply:
That's a good point! Thanks for bringing this up.

I'll add in an explanation to the documentation. How's this sound?

> A truth file represents the output of a workflow for a given input. For example, if your workflow should always output  file named NWD2042242.txt containing the string "NWD2042242" when passed NWD2042242.crai, then your truth file is a file named NWD2042242.txt containing the string "NWD2042242". Now, if you run that same workflow with the same input, but end up with a different output (such as the string being "NWD2042242NWD2042242"), you know something has gone wrong because that output does not match the truth file.

Reply:
Yes, that helps a lot! Thanks